components/org.wso2.carbon.business.rules.core/src/test/java/org/wso2/carbon/business/rules/core/TemplateManagerTest.java

```diff
@@ -1,14 +1,26 @@
 package org.wso2.carbon.business.rules.core;
```

Comment:
What does this class do? 
No any license header as well

Reply:
This was used for testing purposes while doing the implementation, removed through 1dd71e2f12ee08489377cc42b1df7f942286e0ae